e2e-tests/cypress/support/appellant-submission-supporting-documents/confirmSupportingDocumentRejectedBecause.js

```diff
@@ -1,6 +1,6 @@
 module.exports = (errorMessage) => {
   // confirm we are in the right place
-  cy.url().should('include', '/appellant-submission/supporting-documents');
+  cy.url().should('include', '/appeal-householder-decision/supporting-documents');
```

Comment:
please can the dir here be updated to reflect `appeal-householder-decision` - thanks